src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
```

Comment:
What is an example string you'd put into this?
Perhaps this would be better as a boolean with just a true/false value? Something like:
``private boolean hasStem;``

Reply:
I like it... I want to do that!

Reply:
Excellent - go to it! ;)